multi/imxrt-multi/spi.c

```diff
@@ -593,10 +594,13 @@ static void spi_initPins(void)
 
 	for (i = 0; i < sizeof(info) / sizeof(info[0]); ++i) {
 		for (j = 0; j < sizeof(info[i].muxes) / sizeof(info[i].muxes[0]); ++j) {
-			common_setMux(info[i].muxes[j], 0, spi_muxVal(info[i].spi, info[i].muxes[j]));
-
-			if (spi_getIsel(info[i].muxes[j], &isel, &val) == 0)
-				common_setInput(isel, val);
+			/* Check mux value validity. If invalid don't change gpio mux settings */
+			int muxVal = spi_muxVal(info[i].spi, info[i].muxes[j]);
+			if (muxVal >= 0) {
+				common_setMux(info[i].muxes[j], 0, muxVal);
+				if (spi_getIsel(info[i].muxes[j], &isel, &val) == 0)
```

Comment:
np: You could also fix the style to the new one (surround if's body with braces).